referral requests/README.md

```diff
@@ -0,0 +1,173 @@
+                                }
+                            ]
+                        }
+                    ],
+                    "system": "http://snomed.info/sct",
+                    "code": "183885007",
+                    "display": "Private referral to obstetrician"
+                }
+            ],
+            "text": "Reason Code 1"
+        }
+    ],
+    "note": [
+        {
+            "text": "Priority: Routine"
+        },
+        {
+            "text": "Action Date: 2005-04-06"
+        },
+        {
+            "text": "Test request statement text\n                                                            New line\n                                                        "
+        }
+    ]
+}
+```
+</details>
+
+## JSON FHIR > XML HL7
+
+An Allergy Intolerance is mapped to a RequestStatement 
+
+| Mapped to (XML HL7)                                                                                                          | Mapped from (JSON FHIR / other source )                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `RequestStatement / id [@root]`                                                                                              | `ReferralRequest.id` or generated UUID                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
It's a poorly named method, but `getOrNew()` uses the `ReferralRequest.id` to check if the ID is already mapped, however the adaptor will always generate a new UUID. It just checks to see if one has already been generated.

Reply:
Ahh thanks, updated accordingly